src/styles/_settings.scss

```diff
@@ -2,6 +2,13 @@
 $white:      #fff !default;
 $black:      #000 !default;
 
+$header-color: #E5E5E5;
```

Comment:
I like to use $color-1, $color-2, $fon-family-1

I other words $[attr-name]-[number]